src/ebird-ext/readSplitFile.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs')
+const readline = require('readline')
+const csv = require('csv-parse')
+const Town_boundaries = require('./vt_towns.json')
+const Vermont_regions = require('./Polygon_VT_Biophysical_Regions.json')
+const VermontSubspecies = require('./vermont_records_subspecies.json')
+const GeoJsonGeometriesLookup = require('geojson-geometries-lookup')
+const vermontTowns = new GeoJsonGeometriesLookup(Town_boundaries)
+const vermontRegions = new GeoJsonGeometriesLookup(Vermont_regions)
+const eBird = require('./')
+const parser = csv({
+  delimiter: '\t',
+  record_delimiter: '\n',
+  skip_empty_lines: true,
+  relax_column_count: true, // this will cause a blow up if removed
+  relax: true, // this should allow for the double quotes in individual columns, specifically field notes
+  from: 2, // starting on file 53
+  quote: '"',// this also helps to prevent errors on quotes
+  ltrim: true,
+  rtrim: true,
+  columns: [
+    'id',
+    'updated',
+    'order',
+    'category',
+    'commonName',
+    'scientificName',
+    'subspeciesCommonName',
+    'subspeciesScientificName',
+    'count',
+    'breedingCode',
+    'breedingCategory',
+    'ageSex',
+    'country',
+    'countryCode',
+    'state',
+    'stateCode',
+    'county',
+    'countyCode',
+    'iba',
+    'codeusfws',
+    'atlas',
+    'locality',
+    'localityId',
+    'localityType',
+    'wtf',
+    'latitude',
+    'longitude',
+    'observationDate',
+    'timeStarted',
+    'observer',
+    'samplingEventId',
+    'protocolType',
+    'protocolCode',
+    'projectCode',
+    'minutes',
+    'km',
+    'hanumber',
+    'allSpeciesReported',
+    'groupId',
+    'hasMedia',
+    'approved',
+    'reviewed',
+    'reason',
+    'comments',
+    'speciesComments'
+  ]
+})
+
+const filepath = '/Users/benacker/src/birdinginvermont.com/private-data/nosingles.txt'
```

Comment:
Should we perhaps include this as an argument from the CLI?

Reply:
I think that to move it into the CLI would need some refactoring, which I can help with soon (specifically within 3-4 days). I'm 100% okay to do the refactor before merging, too. At least this gets you the functions that chunk the files for a faster dev cycle, and I can perform the refactor and documentation before an actual merge.

Reply:
Sweet. I'll wait up. Wanting to use the chunking to do some cool stuff with towns for a blog for https://vtecostudies.org/. Let me know when you're able to hack at it! :)